views/newsletter/editor.html

```diff
@@ -1340,8 +1340,8 @@ <h1 style="display:none"><%= __('Newsletter Editor') %></h1>
           "newsletter_editor/image-missing.svg"
         ) %>',
       },
-      dragDemoUrl: '<%= image_url("newsletter_editor/editor-drag-demo.mp4") %>',
-      backgroundImageDemoUrl: '<%= image_url("newsletter_editor/background-image-demo.mp4") %>',
+      dragDemoUrl: '//ps.w.org/mailpoet/assets/newsletter-editor/editor-drag-demo.20181121-1440.mp4',
```

Comment:
👍 I didn't really want to introduce new global variables so I used `WP_DEBUG` since it is used that way in many other situations. This should be replaced by stages (development vs. production) but that's much bigger refactoring. Currently there's nothing like that and `WP_DEBUG` is (mis)used often like that.

(Also I used naming `cdn_url` since it seemed more obvious to me what it does in the templates.)

Reply:
`WP_DEBUG` is a WordPress constant, it may be set to `true` on some client websites. It's fine to use it to control caching/logging, but in this particular case it would make CDN-hosted images dissappear in a production build. Let's use something exclusive to our development environment, like an environment variable.

Reply:
Makes sense 👍 I'll add a constant for CDN URL then.

Reply:
It can be a boolean as well, e.g. `MAILPOET_DEVELOPMENT` - like a first step to the stages refactoring.

Reply:
How does `MAILPOET_STAGE` sound to you? That would allow having more than 2 stages (i.e. `production`, `development`, `test`, etc. The only problem is that it's probably not the best thing to do an IF like `if (MAILPOET_STAGE === 'development')` - the cleaner thing to do would be having specific configs for specific stages passed to DI. But as a first step...

Reply:
This could work, though it's often only necessary to distinguish between production and non-production environment with a shortcut like `if (!PRODUCTION)`. Do any way you like since this is a temporary solution until a proper task for stages.

Reply:
Could we not start this `MAILPOET_STAGE` and deciding features based on a stage? It makes configuring the plugin harder. If someone needs only one feature enabled it will make their life harder. 

I was burned by something like this many times in the past. This is opening a pandora box. Because the next guy will configure his feature based on stage as well and then it will become a problem. What is an issue with a single purpose option?

Reply:
I agree there should be no IFs based on STAGE in the code. It should be used only to tell the project from outside which configuration it should load. I'm just thinking which solution we should pick for now. I can add a constant with CDN URL maybe...

Reply:
I just realized that instead of having to type full CDN URL on dev (like localhost:port/...) it's more convenient to just say "use CDN" or "don't use CDN". Therefore I made it configurable like this: https://github.com/mailpoet/mailpoet/pull/1644/commits/072a8a8f6bd571400fa350b193c38a123191e0e1#diff-d122b2bdd8491fd3aa2c9f1bb738c18bR85

Again, this will be nicer to from DI in future but that needs introducing stage configs.

Reply:
Oh, and I have the ternary condition wrong, let me fix it 🤦‍♂️

Reply:
Fixed.